### PR TITLE
add log only

### PR DESCRIPTION
**Pull request by yinwang-wish**

### CODE REVIEW

File: pkg/mongoproxy/plugins/schema/schema.go [hunk 1]
```diff
@@ -122,30 +125,15 @@ func (p *SchemaPlugin) Configure(d bson.D) error {
...
-					return
-				}
-				logrus.Errorf("Schema watcher: %v", err)
-			}
+			p.LoadSchema()
+			time.Sleep(5 * time.Minute)
 		}
 	}()
 
 	if err := watcher.Add(path.Dir(p.conf.SchemaPath)); err != nil {
```
Comment (yiguo-wish) [inline pkg/mongoproxy/plugins/schema/schema.go]: I guess we dont need watcher, can we clean it up from codes?

File: pkg/mongoproxy/plugins/schema/types.go [hunk 2]
```diff
@@ -255,7 +255,8 @@ func (c *Collection) ValidateUpdate(ctx context.Context, obj bson.D, upsert bool
 		}
 	} else {
 		for _, e := range obj {
-			logrus.Debugf("update with operator: %s", e.Key)
+			// TODO: CHANGE TO DEBUGF
+			logrus.Infof("update with operator: %s", e.Key)
```
Comment (yiguo-wish) [inline pkg/mongoproxy/plugins/schema/types.go]: change it to debug?

File: pkg/mongoproxy/plugins/schema/schema.go (hunk 1 above)
Comment (yinwang-wish) [reply pkg/mongoproxy/plugins/schema/schema.go]: fixed

File: pkg/mongoproxy/plugins/schema/types.go (hunk 2 above)
Comment (yinwang-wish) [reply pkg/mongoproxy/plugins/schema/types.go]: fixed

File: pkg/mongoproxy/plugins/schema/schema.go [hunk 3]
```diff
@@ -114,38 +114,18 @@ func (p *SchemaPlugin) Configure(d bson.D) error {
 	if err := p.LoadSchema(); err != nil {
 		return err
 	}
-	// start watch
-	watcher, err := fsnotify.NewWatcher()
+
 	if err != nil {
 		log.Fatal(err)
 	}
```
Comment (yiguo-wish) [inline pkg/mongoproxy/plugins/schema/schema.go]: is this needed?

Comment (yinwang-wish) [reply pkg/mongoproxy/plugins/schema/schema.go]: removed

**Review by yiguo-wish — APPROVED** [review]

**Pull request merged**